lib/versioned/^4.0.0/log/getTask.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var _ = require('lodash');
+var isString = _.isString;
+var isObject = _.isPlainObject;
+var isFunction = _.isFunction;
+
+function getTask(gulpInst) {
+  return function(name) {
+    var task = gulpInst.task(name);
+    var func = isFunction(task.unwrap) ? task.unwrap() : {};
```

Comment:
this can be moved to the `getDescription` and `getFlags` functions.  It will end up calling `unwrap` twice in some scenarios but that shouldn't be much overhead and it avoids it if the properties are on the wrapper